### PR TITLE
[PLAY-745] Implementing React Kit Props

### PR DESCRIPTION
**Pull request by carloslimasd**
**What does this PR do?**
[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-745)

- This PR improves how we show the props available to our kits. We pass the prop name, type, and value into a table. We are only showing the local kit props and not the global props. The Global props will come in a later story.

**Screenshots:** 
image

**How to test?** Steps to confirm the desired behavior:
1. Go to any kit in Playbook
2. Click on the React side and scroll to the bottom
3. Check for the table showing all the kit props

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/config/webpack/environment.js [hunk 1]
```diff
@@ -12,6 +12,20 @@ environment.loaders.insert('javascript', {
...
+    rules: [
+      {
+        test: /\.mjs$/,
+        include: /node_modules/,
+        type: 'javascript/auto',
+      },
+    ],
+  },
+})
+
```
Comment (thestephenmarshall) [inline playbook-website/config/webpack/environment.js:28]: This gets `webpacker` past the `Can't import the named export...` error. By splitting this config out from the `javascript` one, we bypass `babel-loader` which isn't necessary as the source is already transpiled.

File: playbook-website/app/javascript/components/AvailableProps/globalPropsValues.ts [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    values: '"none" | "deep" | "deeper" | "deepest"'
+  },
+  {
+    key: 39,
+    prop: "space",
+    type: "union",
+    values: '"spaceBetween" | "spaceAround" | "spaceEvenly"'
+  },
+  {
+    key: 40,
```
Comment (jasoncypret) [inline playbook-website/app/javascript/components/AvailableProps/globalPropsValues.ts:243]: Why key? I imagine we could add more props in above and cause this key to be 41 for example.

File: playbook-website/app/javascript/site_styles/_kits_show.scss [hunk 3]
```diff
@@ -17,4 +18,11 @@
       align-self: center;
     }
   }
+  .kearning {
+    letter-spacing: $lspace_looser;
+  }
+  .card {
+    max-width: fit-content;
+    display: inline-block;
+  }
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_kits_show.scss:27]: Anything we can do to use global props and eliminate the need for CSS?

File: playbook-website/app/views/pages/kit_show.html.erb [hunk 4]
```diff
@@ -94,5 +94,11 @@
   <% if (action_name == "kit_show_rails") %>
     <br>
     <%= pb_rails("docs/kit_api", props: { kit: @kit }) %>
+  <% else %>
+    <br>
+    <%= pb_rails("title", props: { text: "Available Props", size: 3, margin_bottom: "sm" }) %>
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/kit_show.html.erb:99]:
```suggestion
    <%= pb_rails("title", props: { text: "Available Props", size: 3, margin_bottom: "sm", margin_top: "sm" }) %>
```

Lets try and avoid br tags

File: playbook-website/app/javascript/site_styles/_kits_show.scss (hunk 3 above)
Comment (jasperfurniss) [reply playbook-website/app/javascript/site_styles/_kits_show.scss:27]: @nidaqg is trying a few things out now on this one.

**Review by jasperfurniss — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**